server/api/utils/builder.py

```diff
@@ -400,6 +391,14 @@ def build_image(
     runtime_spec = runtime.spec if runtime else None
     runtime_builder_env = runtime_spec.build.builder_env or {}
 
+    project_default_function_node_selector = {}
```

Comment:
use the helper here?

Reply:
I pushed the helper by mistake, we said we won't add it cause it needs the runtime, I'm removing it now